src/test/rustdoc/issue-76501.rs

```diff
@@ -0,0 +1,18 @@
+#![feature(const_fn)]
+
+// @has 'issue_76501/fn.bloop.html' '//pre' 'pub const fn bloop() -> i32'
+/// A useless function that always returns 1.
+pub const fn bloop() -> i32 {
+    1
+}
+
+/// A struct.
+pub struct Struct {}
+
+impl Struct {
+    // @has 'issue_76501/struct.Struct.html' '//*[@class="method"]' 'pub const fn bloop() -> i32'
+    /// A useless function that always returns 1.
+    pub const fn bloop() -> i32 {
+        1
+    }
+}
```

Comment:
Can you also add a test for trait items?
```suggestion
}

trait T {
    // @has 'issue_76501/trait.T.html' '//*[@class="method"]' 'pub const fn f()'
   const fn f();
}
```

Reply:
That isn't valid code.

`error[E0379]: functions in traits cannot be declared const`

For const trait methods, you'd just declare the methods of a trait `Foo` normally, and then if using `#![feature(const_trait_impl)]` you could do `impl const Foo for Bar {}`.

Reply:
Oh, right. Ignore me then.